lib/blowocking/Cargo.toml

```diff
@@ -1,14 +1,14 @@
 [package]
-name = "kitsune-blocking"
+name = "blowocking"
 authors.workspace = true
 edition.workspace = true
 version.workspace = true
-license.workspace = true
+license = "MIT OR Apache-2.0"
 
 [dependencies]
 once_cell = "1.19.0"
 rayon = "1.8.1"
-thiserror = "1.0.56"
+thiserror = "1.0.57"
 tokio = { version = "1.36.0", features = ["rt", "sync"] }
```

Comment:
The `rt` feature should probably be made optional by gating the I/O blocking function behind a feature flag.

For the `http-signatures` crate we only need the `sync` feature